packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-misc--to-es2015/super/exec.js

```diff
@@ -0,0 +1,24 @@
+function decorate() {
+  return function (target, context) {}
+}
+
+class Test {
+  @decorate()
+  accessor foo = 42;
+
+  constructor() {
+  }
+}
+
+new Test()
+
+class TestChild extends Test {
+  @decorate()
+  accessor bar = 1;
+
+  constructor() {
+    super();
+  }
+}
+
+const r = new TestChild();
```

Comment:
In contrast with `initProto-existing-derived-constructor`, can you rename this test to `initField-existing-derived-constructor`?

I suggest we also add some behaviour assertions like we have done in that test.

Reply:
I tried porting all the tests but unfortunately couldn't get them all to work so I just added a simple test.🤦‍♂️

Reply:
Can you push the new tests? Even if they are failing, we can still investigate for sure.

Reply:
![image](https://github.com/babel/babel/assets/30521560/b55ea4d5-4388-4b8c-8533-f8548cc16ff8)
I pushed it, but it doesn't show up on github, I will try force pushing.🤔